### PR TITLE
Adds schedule and chain policy to fleetautoscaler documentation

### PR DESCRIPTION
**Pull request by indexjoseph**
**What type of PR is this?**

/kind documentation

**What this PR does / Why we need it**:

Adds schedule/chain policy to the fleetautoscaler documentation on the agones site

**Which issue(s) this PR fixes**:

Towards #3008

**Special notes for your reviewer**:

### CODE REVIEW

File: site/content/en/docs/Reference/fleetautoscaler.md
Comment (vicentefb) [inline site/content/en/docs/Reference/fleetautoscaler.md:179]: I think we might need to add a block like the one mentioned here https://agones.dev/site/docs/contribute/#within-a-page so that it renders in the version we want.  v1.43.0 (?)

Comment (indexjoseph) [reply site/content/en/docs/Reference/fleetautoscaler.md:179]: I see, since it hasn't been realesed yet, sg!

Comment (igooch) [reply site/content/en/docs/Reference/fleetautoscaler.md:179]: It will be rolled out in 1.43.0 which gets released tomorrow.

File: site/content/en/docs/Reference/fleetautoscaler.md
Comment (igooch) [inline site/content/en/docs/Reference/fleetautoscaler.md:182]: Nit: "A scheduled based ... based on a schedule" could be reworded. Something like: A schedule-based autoscaler can automatically scale GameServers at a specific time based on a FleetAutoscaler policy.

File: site/content/en/docs/Reference/fleetautoscaler.md
Comment (igooch) [inline site/content/en/docs/Reference/fleetautoscaler.md:229]: Nit: Add a comment on how the index of each policy within the list determines the priority. I.e. does the first item in the list have the highest priority?

File: site/content/en/docs/Reference/fleetautoscaler.md
Comment (igooch) [inline site/content/en/docs/Reference/fleetautoscaler.md:231]: Nit:
```suggestion
For example, if you have an in-game event happening between 12:00 AM and 10:00 PM on October 31, 2024 EST, and you want to ensure there are always at least 5 ready game servers during that time, you chain add a schedule policy that applies a buffer policy with a buffer size of 5. Now, let's say you want to adjust the number of game servers after the event based on external factors after the in-game event, you could chain a webhook policy to dynamically. Finally as a fallback, for if the webhook invocation fails, you can chain another buffer policy to default to.
```

File: site/content/en/docs/Reference/fleetautoscaler.md
Comment (igooch) [inline site/content/en/docs/Reference/fleetautoscaler.md:235]: I would move this note up to just after the scheduled autoscaler example. That way the chain autoscaler explanation is right next to the associated example.

**Review by igooch — COMMENTED** [review]
Looks good, just a few nits.

**Review by igooch — APPROVED** [review]
LGTM

**Comment by igooch** [comment]
We'll wait until after the release tomorrow to merge.

**Pull request merged**
